include/mbedtls/rsa.h

```diff
@@ -403,6 +403,16 @@ void mbedtls_rsa_set_padding( mbedtls_rsa_context *ctx, int padding,
  */
 size_t mbedtls_rsa_get_len( const mbedtls_rsa_context *ctx );
 
+/**
+ * \brief          This function retrieves the length of the RSA modulus in bits.
+ *
+ * \param ctx      The initialized RSA context.
+ *
+ * \return         The length of the RSA modulus in bits.
+ *
+ */
+size_t mbedtls_rsa_get_bitlen( const mbedtls_rsa_context *ctx );
```

Comment:
Unfortunately adding this function breaks existing ALT implementations since they don't provide it. Furthermore I don't like adding this extra function without adding a few more tests.